apps/frontend/src/app/3_organisms/ProposalVotingResults/index.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FC, useMemo } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Bar,
+  Button,
+  ButtonStyle,
+  HelperButton,
+  Paragraph,
+  ParagraphSize,
+} from '@sovryn/ui';
+import { Decimal } from '@sovryn/utils';
+
+import { useBlockNumber } from '../../../hooks/useBlockNumber';
+import { translations } from '../../../locales/i18n';
+import { Proposal } from '../../../utils/graphql/rsk/generated';
+import { decimalic, formatValue } from '../../../utils/math';
+
+export type ProposalVotingResultsProps = {
+  proposal: Proposal;
+  onExportCSV?: () => void;
+};
+
+export const ProposalVotingResults: FC<ProposalVotingResultsProps> = ({
+  proposal,
+  onExportCSV,
+}) => {
+  const currentBlock = useBlockNumber().value;
+  const { votesFor, votesAgainst, endBlock, emittedBy, quorum } = proposal;
+  const { majorityPercentageVotes, quorumPercentageVotes } = emittedBy;
+
+  const { support, turnout, votes } = useMemo(() => {
+    const votes = Decimal.fromBigNumberString(votesFor).add(
+      Decimal.fromBigNumberString(votesAgainst),
+    );
+
+    const support = Decimal.fromBigNumberString(votesFor)
+      .div(votes)
+      .mul(100)
+      .toNumber();
+
+    const turnout = votes
+      .div(
+        Decimal.fromBigNumberString(quorum).div(
+          decimalic(quorumPercentageVotes).div(100),
+        ),
+      )
+      .mul(100)
+      .toNumber();
+
+    return {
+      votes,
+      support,
+      turnout,
+    };
+  }, [quorum, quorumPercentageVotes, votesAgainst, votesFor]);
+
+  return (
+    <div className="w-full bg-gray-90 p-6">
+      <div className="flex flex-row justify-between mb-8">
+        <Paragraph size={ParagraphSize.base} className="font-medium">
+          {t(translations.proposalVotingResults.title)}
+        </Paragraph>
+        {currentBlock < endBlock ? (
+          <Paragraph size={ParagraphSize.small} className="italic">
+            {t(translations.proposalVotingResults.inProgress)}
```

Comment:
when proposal in progress, the title should be `In progress, Will succeed` or `In progress, Will be rejected`
<img width="348" alt="inProgress" src="https://github.com/DistributedCollective/sovryn-dapp/assets/74987028/3526e73a-255f-4269-bbae-bbf74c249ecc">